### DISCUSSION TITLE
Is there a way to get tabs showing all the time?

### DISCUSSION THREAD

**Discussion by arminolat** [aleksey-hoffman/sigma-file-manager#87] [Q&A] [answered]
Hello!

I downloaded the program today and so far so good!

That said, I'm wondering if there's any way to show tabs in the top of the program without using keyboard shortcuts, like in web browsers?
Also, is there a way to have split screen two folders, so I can just drag and drop files?

Finally, is there a way to turn off animations? They are slightly distracting. 

Thanks!

**Comment by Ansh4011** [comment, marked as answer]
Hey, I am not the developer, but I somehow know this, so-->
Well, for the tabs, not yet buddy, but yes, this is another future update, coming soon in the next few months probably I guess! 
And that for the drag and drop, you can use tabs and even workspace if you need! And also, using right click context menu you can keep it in the clipboard and easily transfer it in just one click!
Animations maybe get tweaked in the coming future, but it will be tweaked for sure as far as I know!
*This all just as far as I know!

**Comment by Ansh4011** [comment]
Also, you can refer to the code readme.md!
